File: pgm/ABC_C#/ABC_156_A.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int N = int.Parse(arr[0]), R = int.Parse(arr[1]);
        Console.WriteLine(10 <= N ? R : R + 100 * (10 - N));
    }
}

File: pgm/ABC_Visual_Basic/ABC_156_A.vb
Class Main
    Public Shared Sub Main()
        Dim N, R As Integer
        Dim arr(2) As String
        arr = Split(Console.ReadLine())
        N = arr(0) : R = arr(1)
        Console.WriteLine(if(10 <= N, R, R + 100 * (10 - N)))
    End Sub
End Class
